examples/python/air_traffic_data/air_traffic_data.py

```diff
@@ -310,7 +315,13 @@ def log_position_and_altitude(self, df: polars.DataFrame, icao_id: str) -> None:
             return
 
         if icao_id not in self._position_indicators:
-            rr.log(entity_path, [rr.archetypes.Points3D.indicator(), rr.archetypes.GeoPoints.indicator()], static=True)
+            color = rr.components.Color.from_string(entity_path)
+            rr.log(
+                entity_path,
+                [rr.archetypes.Points3D.indicator(), rr.archetypes.GeoPoints.indicator(), color],
```

Comment:
Good lord I didnt realize how much that code needs tagged components :see_no_evil:

Reply:
Good example of a component which is _intended_ to be shared across two archetypes, btw. Hopefully we don't make that use case too contrived, aka if that `Color` is  tagged with the `Points3D` archetype, it should be implicitly(?) used by the `GeoPoints` archetype as well.

Reply:
Ha yeah I didn't even realize that was what's happening, interesting.

Yeah we'll have to figure out how we want to handle those. I do think heuristics to implicitly pick up the same component from unrelated tags if on the same entity makes sense.

Reply:
Vaguely related is one realisation we've had when discussing the archetype vs. visualiser relationship the other day: indicator components are additive, while tags are exclusive. With the former, you could have 2 archetypes with 1 components (e.g. hypothetical ScalarSeriesLine + ScalarSeriesPoint archetypes based on a single scalar component + 2 indicators), but with the latter you are forced to make a choice.

We don't think this constraint is going to be much of problem though. More in https://github.com/rerun-io/rerun/issues/8368